.ruff.toml

```
@@ -203,6 +202,7 @@ lint.unfixable = [
 "astropy/constants/*" = []
 "astropy/convolution/*" = []
 "astropy/coordinates/*" = [
+    "B007",  # UnusedLoopControlVariable
```

Please use the easily searchable id [used in ruff's documentation](https://docs.astral.sh/ruff/rules/unused-loop-control-variable/), here and elsewehere
```suggestion
    "B007",  # unused-loop-control-variable
```

---

I remove B0007 under coordinates in ruff.toml. because pull request astrpy/coordinates has been merged